Dockerfile.dev

```diff
@@ -1,2 +1,3 @@
 FROM runatlantis/atlantis:latest
 COPY atlantis /usr/local/bin/atlantis
+WORKDIR /go/src/github.com/runatlantis/atlantis
```

Comment:
Is this needed with the latest golang versions?

Reply:
This was just preference, I wanted to be able to exec into the container and have a shell right where my src code is.